Classes/Controllers/PBGitCommitController.m

```diff
@@ -164,26 +173,39 @@ - (IBAction) forceCommit:(id) sender
 - (void) commitWithVerification:(BOOL) doVerify
 {
 	if ([[NSFileManager defaultManager] fileExistsAtPath:[repository.gitURL.path stringByAppendingPathComponent:@"MERGE_HEAD"]]) {
-		[[repository windowController] showMessageSheet:@"Cannot commit merges" infoText:@"GitX cannot commit merges yet. Please commit your changes from the command line."];
+		[repository.windowController
+		 showMessageSheet:NSLocalizedString(@"Cannot commit merges",
+											@"Title for sheet that GitX cannot create merge commits")
+		 infoText:NSLocalizedString(@"GitX cannot commit merges yet. Please commit your changes from the command line.",
+									@"Information text for sheet that GitX cannot create merge commits")];
 		return;
 	}
 
 	if ([[cachedFilesController arrangedObjects] count] == 0) {
-		[[repository windowController] showMessageSheet:@"No changes to commit" infoText:@"You must first stage some changes before committing"];
+		[repository.windowController
```

Comment:
Adjacent string literals get concatenated, if that helps:

```objc
NSLocalizedString(@"You need to stage some changed files before committing "
                   "by moving them to the list of Staged Changes.",
                  @"Information text for sheet that you need to "
                   "stage changes before creating a commit")
```

The @ is optional on subsequent NSString literals.